.gitignore

```diff
@@ -1,3 +1,5 @@
 web/
 docker-compose.override.yml
 drupal.make
+.idea
```

Comment:
Please do not add `.idea` to project-specific `.gitignore`. If you use IntelliJ it belongs in [your global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
👍